app/helpers/registration_wizard_helper.rb

```diff
@@ -6,4 +6,13 @@ def registration_wizard_form_url(form)
       registration_wizard_update_path
     end
   end
+
+  def title_locale_type(question_type)
+    case question_type
+    when :radio_button_group
+      :legend
+    else
+      :label
+    end
+  end
```

Comment:
@peteryates You may know of a better solution to this, this is accounting for how some questions are stored as legends and others as labels and these need to be gathered during title setup in _question_page.html.erb.